.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: generic
+
+matrix:
+  include:
+    - name: 'Conda Python with Specfile'
+      install:
+        - sudo apt-get update && sudo apt-get install -y wget
+        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+        - bash miniconda.sh -b -p $HOME/miniconda
+        - source "$HOME/miniconda/etc/profile.d/conda.sh"
+        - hash -r
+        - conda config --set always_yes yes --set changeps1 no
+        - conda update -q conda
+        - conda create --name simple_network_sim -f spec-file.txt
```

Comment:
I think the arg here is `--file` not `-f`. I think `-f` only exists for `env create`

Reply:
Pushed a new version with `--file`, and that at least definitely works - see https://travis-ci.com/github/WPettersson/simple_network_sim/builds/166876200